src/utils/uniqueId.js

```diff
@@ -0,0 +1,7 @@
+let id = -1;
+const uniqueId = prefix => {
+  id += 1;
+  return prefix ? `${prefix}${id}` : id;
+};
+
+export default uniqueId;
```

Comment:
I know that's how lodash does it, but now looking at it I'm wondering if it wouldn't break on server side.

Reply:
What do you mean?

Reply:
References:
- https://github.com/FormidableLabs/victory/issues/664
- https://stackoverflow.com/questions/50231301/prop-mismatch-server-client-how-to-generate-a-unique-id-that-persists
- https://github.com/facebook/react/issues/5867